keyboards/zygomorph/keymaps/default/keymap.c

```diff
@@ -0,0 +1,191 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
+    _COLEMAK,
+    _FN,
+    _ADJ
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  FN,
+  ADJ,
+  RGBRST
+};
+
+#define FN_CAPS  LT(_FN, KC_CAPS)
+
+// Define your non-alpha grouping in this define's LAYOUT, and all your BASE_LAYERS will share the same mod/macro columns
+  /*  / Base Layout \
+   * /-----------------------------------------\  /-----------------------------------------\
+   * | GESC |  1   |  2   |  3   |  4   |  5   |  |  6   |  7   |  8   |  9   |  0   | BkSp |
+   * |------+------+------+------+------+------|  |------+------+------+------+------+------|
+   * | Tab  |      |      |      |      |      |  |      |      |      |      |      |   \  |
+   * |------+------+------+------+------+------|  |------+------+------+------+------+------|
+   * |FNCAPS|      |      |      |      |      |  |      |      |      |      |      |   '  |
+   * |------+------+------+------+------+------|  |------+------+------+------+------+------|
+   * |Shift |      |      |      |      |      |  |      |      |      |      |      |Shift |
+   * |------+------+------+------+------+------|  |------+------+------+------+------+------|
+   * | Ctrl |  Win |  Alt |  RGB | ADJ  | Space|  | Space|  FN  | Left | Down | Up   |Right |
+   * \------+------+------+------+------+------/  \------+------+------+------+------+------/
+   */
+#define BASE_LAYOUT( \
```

Comment:
Honestly, I'm not sure that it's a good idea to have this in the default layout, since it may be confusing to users.

Reply:
Possibly, this is also how all the other RGBKB boards have been setup. Also might be getting high time to move them into an rgbkb folder too.

Reply:
Well, those should probably be fixed then, too.   But that can be done later.   

ALso the non-default keymaps are fine, but ... any complexity for the default keymap has usually been greeted with confusion and problems by newer users.  Hence my concern.

And yeah, we should move them to an rgbkb folder.  Though, probably a separate PR, after this one.

Reply:
Did a cleanup pass on the layouts and make files based on all the suggestions. I have not changed the default BASE_LAYOUT macros as @Legonut uses the Default keymaps as his own and that is how he likes in. I'll talk with him about switching these, adding Legonut keymaps, and moving Zen, Sol, & Zygomorph into a rgbkb directory and do that work in a separate pr. Sound good?

Reply:
Perfect.


For now, I think I'll merge the changes, as is. 

But later, a good "cleanup" PR, to fix the above, and/or move all the boards into a folder would be awesome, and the best course, I think.